src/types.ts

```diff
@@ -848,6 +849,12 @@ export interface DraxListProps<TItem> extends Omit<FlatListProps<TItem>, 'render
 	/** Callback handler for when a list item is moved within the list, reordering the list */
 	onItemReorder?: DraxListOnItemReorder<TItem>;
 
+	/** Callback handler for when list is scrolled */
+	onScrollProp?: (scrollEvent: NativeScrollEvent) => void;
```

Comment:
The name here should be `onScroll` rather than `onScrollProp`, but regardless it should not be necessary. `DraxListProps<TItem>` extends `Omit<FlatListProps<TItem>, 'renderItem'>` and passes non-Drax-specific props on to the underlying `FlatList`, so `onScroll` already exists. We just need to call it in our own `onScroll` implementation.